docs/source/examples/Widget List.ipynb

```diff
@@ -537,12 +537,34 @@
    "source": [
     "widgets.RadioButtons(\n",
     "    options=['pepperoni', 'pineapple', 'anchovies'],\n",
-    "#     value='pineapple',\n",
+    "#    value='pineapple', # Defaults to 'pineapple'\n",
+    "#    layout={'width': 'max-content'}, # If the items' names are long\n",
     "    description='Pizza topping:',\n",
     "    disabled=False\n",
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### With dynamic layout, no width restriction"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "widgets.Box(\n",
+    "    [\n",
+    "        widgets.Label(value='Pizza topping with a very long label:'), \n",
+    "        widgets.RadioButtons(options=['pepperoni', 'pineapple', 'anchovies', 'and the long name that will fit fine'])\n",
```

Comment:
Do you want `layout={'width': 'max-content'}` somewhere in here? For me, this text fits fine with the default width, so maybe make the text a little longer so it's clear that it is working?

Reply:
That's the whole point; if you use boxes (custom layout), you don't need to set the `width` at all.

Reply:
> That's the whole point; if you use boxes (custom layout), you don't need to set the `width` at all.

It doesn't work for me with the latest widgets release, though I have to make the string a bit longer to see it:
<img width="1071" alt="Screen Shot 2020-01-14 at 3 52 03 AM" src="https://user-images.githubusercontent.com/192614/72310281-a4b18580-3681-11ea-9b06-91d1b4c9f154.png">

Reply:
Great catch, thanks! It seems like it will fit in more, but not all of the text. I added the same layout changes to the Box example.